crates/aiken-lang/src/error/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub trait ExtraData {
```

Comment:
fancy

Reply:
Kind of hacky for now but sufficient for the few quick fixes I needed to implement. Ideally, I'd like the data to be some serialisable thing or even just a serde_json::Value to make it more flexible. But then that means introducing serde as a dependency of Aiken-lang and I didn't feel quite like doing it now.